api/submission/fetch_attempts.go

```diff
@@ -0,0 +1,42 @@
+package submission
+
+import (
+    "github.com/eriq-augustine/autograder/api/core"
+    "github.com/eriq-augustine/autograder/db"
+    "github.com/eriq-augustine/autograder/model"
+)
+
+type FetchAttemptsRequest struct {
+    core.APIRequestAssignmentContext
+    core.MinRoleGrader
+
+    TargetUser core.TargetUserSelfOrGrader `json:"target-email"`
+}
+
+type FetchAttemptsResponse struct {
+    FoundUser        bool `json:"found-user"`
```

Comment:
Spacing here looks a little strange.
Should it align with the open bracket on the line below?